test/test_nn.py

```diff
@@ -1689,13 +1693,13 @@ def test_orthogonal(self):
                         input_tensor = input_tensor.data
 
                     rows, cols = tensor_size[0], int(np.prod(tensor_size[1:]))
-                    flattened_tensor = input_tensor.view(rows, cols).numpy()
+                    flattened_tensor = input_tensor.view(rows, cols)
                     if rows > cols:
-                        assert np.allclose(np.dot(flattened_tensor.T, flattened_tensor), np.eye(cols) * gain ** 2,
-                                           atol=1e-6)
+                        assert torch.dist(torch.mm(flattened_tensor.t(), flattened_tensor),
```

Comment:
I think you can pass a tolerance parameter to `self.assertEqual` that handles tensors as well, see https://github.com/pytorch/pytorch/blob/master/test/common.py#L93